PinballY/SecondaryView.cpp

```diff
@@ -471,6 +473,15 @@ void SecondaryView::StartBackgroundCrossfade()
 	incomingBackground.sprite->StartFade(1, crossFadeTime);
 }
 
+void SecondaryView::EndAnimation()
+{
+	if (incomingBackground.sprite != nullptr && !incomingBackground.sprite->IsFadeDone()) {
+		incomingBackground.sprite->EndFade();
+		incomingBackground.sprite->UpdateFade();
+		UpdateAnimation();

```

Comment:
This is obviously a bit of a hack. It's working, but I'm positive there's a cleaner solution